src/ring_theory/polynomial/symmetric.lean

```diff
@@ -0,0 +1,166 @@
+/-
+Copyright (c) 2020 Hanting Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Hanting Zhang, Johan Commelin
+-/
+import tactic
+import data.mv_polynomial.rename
+import data.mv_polynomial.comm_ring
+
+/-!
+# Symmetric Polynomials and Elementary Symmetric Polynomials
+
+This file defines symmetric polynomials and elementary symmetric polynomials.
+We also prove some basic facts about them.
+
+## Main declarations
+
+* `mv_polynomial.is_symmetric`
+
+* `mv_polynomial.esymm`
+
+## Notation
+
++ `esymm σ R n`, is the `n`th elementary symmetric polynomial in `mv_polynomial σ R`.
+
+As in other polynomial files, we typically use the notation:
+
++ `σ τ α : Type*` (indexing the variables)
+
++ `R S : Type*` `[comm_semiring R]` `[comm_semiring S]` (the coefficients)
+
++ `s : σ →₀ ℕ`, a function from `σ` to `ℕ` which is zero away from a finite set.
+This will give rise to a monomial in `mv_polynomial σ R` which mathematicians might call `X^s`
+
++ `r : R` elements of the coefficient ring
+
++ `i : σ`, with corresponding monomial `X i`, often denoted `X_i` by mathematicians
+
++ `p : mv_polynomial σ α`
```

Comment:
This seems to be using `alpha` as ring of coefficients, which contradicst what is written on L28.